{
  "uid" : "3c53a757837aa74a",
  "name" : "Failure login to D3O with invalid password",
  "fullName" : "D3O Login: Failure login to D3O with invalid password",
  "historyId" : "f389c2653a05dcf25b1d23626b158805",
  "time" : {
    "start" : 1630839453665,
    "stop" : 1630839463078,
    "duration" : 9413
  },
  "status" : "broken",
  "statusMessage" : "TypeError: __call__() got an unexpected keyword argument 'type'\n",
  "statusTrace" : "  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\d3aloginsteps.py\", line 64, in assert_login_unsuccessful\n    allure.attach('screenshot3', context.driver.get_screenshot_as_png(), type=allure.attachment_type.PNG)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "TypeError: __call__() got an unexpected keyword argument 'type'\n",
    "statusTrace" : "  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\d3aloginsteps.py\", line 64, in assert_login_unsuccessful\n    allure.attach('screenshot3', context.driver.get_screenshot_as_png(), type=allure.attachment_type.PNG)\n",
    "steps" : [ {
      "name" : "Given the user launches Chrome Browser",
      "time" : {
        "start" : 1630839453666,
        "stop" : 1630839455229,
        "duration" : 1563
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "When the user opens D3a login page url \"https://www.d3a.io/login\"",
      "time" : {
        "start" : 1630839455230,
        "stop" : 1630839458538,
        "duration" : 3308
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "When the user enters username \"laks.kswamy@gmail.com\" and password \"wrongPassword\"",
      "time" : {
        "start" : 1630839458538,
        "stop" : 1630839458816,
        "duration" : 278
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "And the user clicks the Login Button",
      "time" : {
        "start" : 1630839458816,
        "stop" : 1630839462900,
        "duration" : 4084
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "Then the home page does not open with invalid password",
      "time" : {
        "start" : 1630839462900,
        "stop" : 1630839463078,
        "duration" : 178
      },
      "status" : "broken",
      "statusMessage" : "TypeError: __call__() got an unexpected keyword argument 'type'\n",
      "statusTrace" : "  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\d3aloginsteps.py\", line 64, in assert_login_unsuccessful\n    allure.attach('screenshot3', context.driver.get_screenshot_as_png(), type=allure.attachment_type.PNG)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : true,
      "attachmentsCount" : 0,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : true,
    "stepsCount" : 5
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "D3O Login"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "3c53a757837aa74a.json",
  "parameterValues" : [ ]
}